_posts/2016-12-05-integrated-shrinking.md

```diff
@@ -155,7 +155,7 @@ your shrinkers, which is more work and more brittle than just having it work
 automatically.
 
 So although (as far as I know) none of the currently popular property based
-testing langugages for statically typed languages implement this behaviour
+testing languages for statically typed languages implement this behaviour
```

Comment:
I think languages is just the wrong word altogether here actually. I guess I meant systems?